doc/man/pam_pwquality.8.pod

```diff
@@ -35,7 +35,7 @@ Is the new password a palindrome?
 
 =item Case Change Only
 
-Is the new password the the old one with only a change of case?
+Is the new password the old one with only a change of case?
```

Comment:
maybe `same as the old one`?

Reply:
Dear @t8m,

Thank you for your comment.

The meaning of the sentence has not changed.
Since there are two "the", I changed it to one.

Reply:
I know. But the sentence would sound better if it was: `Is the new password the same as the old one with only a change of case?` Could you please change it that way?

Reply:
I'm sorry my understanding is wrong.
I fixed it as you suggested.
Would you please review it again?